RNTester/js/PickerExample.js

```diff
@@ -21,10 +21,6 @@ const {Picker, Text} = ReactNative;
 const Item = Picker.Item;
 
 class PickerExample extends React.Component<{}, $FlowFixMeState> {
```

Comment:
This change will conflict with https://github.com/facebook/react-native/pull/22828 but should be the only conflict